### PR TITLE
[BPK-1345] Prepare components for new docs

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: packages/bpk-mixins/src/mixins/_typography.scss [hunk 1]
```diff
@@ -624,6 +645,18 @@
   }
 }
 
+/// Alternate code style suited for display on non-white backgrounds.
+///
+/// @example scss
+///   .selector {
+///     @include bpk-code();
+///     @include bpk-code--alternate();
+///   }
```
Comment (matthewdavidson) [inline packages/bpk-mixins/src/mixins/_typography.scss]: nit: This should have a newline.

File: packages/bpk-mixins/src/mixins/_typography.scss [hunk 2]
```diff
@@ -856,3 +904,34 @@
...
+///
+/// @require {mixin} bpk-content-container
+/// @require {mixin} bpk-content-container--bare-html
+///
+/// @example scss
+///   .selector {
+///     @include bpk-content-container();
+///     @include bpk-content-container--bare-html();
+///     @include bpk-content-container--bare-html-alternate();
+///   }
```
Comment (matthewdavidson) [inline packages/bpk-mixins/src/mixins/_typography.scss]: nit: newline

File: packages/bpk-component-table/readme.md [hunk 3]
```diff
@@ -55,4 +55,6 @@ export default () => (
 
 | Property | PropType | Required | Default Value |
 | -------- | -------- | -------- | ------------- |
-| children | -        | true     | -             |
+| children | node     | true     | -             |
+
+Additionally `BpkTableHeadCell` as a boolean `alternate` prop which is inteded to be use when a table sits on a non-white background.
```
Comment (tiagoengel) [inline packages/bpk-component-table/readme.md]: has a boolean?

Comment (k0nserv) [reply packages/bpk-component-table/readme.md]: Yeah that ain't right. Good catch

**Comment by k0nserv** [comment]
@matthewdavidson here you go. Can I get a new ✅?


image

**Pull request merged**